include/kodlab_mjbots_sdk/joint_moteus.h

```diff
@@ -129,20 +157,43 @@ class JointMoteus: public JointBase
               can_id_( config.can_id),
               can_bus_( config.can_bus),
               moteus_kp_(config.moteus_kp),
-              moteus_kd_(config.moteus_kd){}
+              moteus_kd_(config.moteus_kd),
+              temperature_(config.temperature),
+              q_current_(config.q_current),
+              d_current_(config.d_current),
+              voltage_(config.voltage) {}
         
         /**
-         * @brief Update the joint of the moteus. Converts rot/s to rad/s and saves mode
+         * @brief Update the joint of the moteus. Converts rot/s to rad/s and saves registers
          * 
          * @param reply_pos position reported by moteus [rot]
          * @param reply_vel velocity reported by moteus [rot/s]
          * @param reply_torque torque measured by the moteus [Nm]
          * @param mode      moteus mode
+         * @param reply_q_current current in the Q phase reported by moteus [A]
+         * @param reply_d_current current in the D phase reported by moteus [A]
+         * @param reply_voltage voltage reported by the moteus [V]
+         * @param reply_temperature temperature reported by the moteus [C]
+         * @param fault     fault code
          */
-        void UpdateMoteus(float reply_pos, float reply_vel, float reply_torque, ::mjbots::moteus::Mode mode ){
-            UpdateState( 2 * M_PI * reply_pos, 2 * M_PI * reply_vel, reply_torque);
-            mode_ = mode;
-        }
+        void UpdateMoteus(
+            float reply_pos,
+            float reply_vel,
+            float reply_torque,
+            ::mjbots::moteus::Mode mode,
+            float reply_q_current,
+            float reply_d_current,
+            float reply_voltage,
+            float reply_temperature,
+            ::mjbots::moteus::Fault fault){
```

Comment:
Should we just take in the reply itself and parse some here?

Reply:
That really depends on how we are scoping, and I think both options are reasonable.